presto-orc/src/main/java/com/facebook/presto/orc/metadata/Stream.java

```diff
@@ -21,21 +21,38 @@
 
 public class Stream
 {
+    public enum StreamArea {
```

Comment:
Added a new enum to indicate the area where a stream kind belongs to to be able to easily distinguish INDEX streams without adding a multi-layer condition.